Moderator & label triggers for new labels

mdanish-kh opened the pull request:
cc @Trenly @stephengillie

PR adds moderator triggers for:

- Last-Version-Remaining
- License-Blocks-Install
- Scripted-Application
- DriverInstall
- Network-Blocker

Label trigger is added for

- Last-Version-Remaining
- License-Blocks-Install

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/137512)

mdanish-kh committed:
Moderator and label triggers for new labels

Trenly reviewed (commented):
This label should probably also be removed on changes pushed (Should be in the `issueUpdated.yml` under the `isAction: synchronize`) and maybe also on Reset Feedback?

Trenly commented:
Could I also make a request for mod triggers on https://github.com/microsoft/winget-pkgs/labels/License-Blocks-Install , https://github.com/microsoft/winget-pkgs/labels/Scripted-Application , https://github.com/microsoft/winget-pkgs/labels/DriverInstall , and https://github.com/microsoft/winget-pkgs/labels/Network-Blocker 

I think these are new ones @stephengillie added recently

mdanish-kh commented:
@Trenly - 

I've added the moderator triggers for the new labels. Also added a label trigger for the `License-Blocks-Install`. Could add for the others, but I'm not 100% sure on what triggers them and the context around those labels.

Last-Version-Remaining is removed when new commits are pushed. Didn't add for the other new ones as pipelines don't have a way to automatically re-apply them, but added option for removing them through [Policy] reset feedback

mdanish-kh committed:
wording
